### PR TITLE
feat : added navbar

### PR DESCRIPTION
**Pull request by MurtazaBagwala07**
added a basic navbar which consists of title , and other liks to diff pages

### CODE REVIEW

File: components/navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.navbar-container{
+    width: 100%;
+    display:flex;
+    flex-direction: row;
```
Comment (shrey27) [inline components/navbar/navbar.css]: just a suggestion. I you are using same properties more than once like 
display:flex;
align-items:center etc. better put it in a separate class.
But you want to add responsiveness to your page then it is fine.

**Review by shrey27 — APPROVED** [review]

File: components/navbar/navbar.css (hunk 1 above)
Comment (lomesshh) [inline components/navbar/navbar.css]: flex-direction is by default set to row so you can remove this property

File: components/navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    font-size:2rem;
+    margin:0.25rem 1rem;
+    border-radius: 0.25rem;
+}
+
+.navbar-links{
+    list-style: none;
+    margin-left: auto;
+    display:flex;
+    flex-direction: row;
```
Comment (lomesshh) [inline components/navbar/navbar.css]: remove flex direction

**Review by lomesshh — COMMENTED** [review]
do those two changes other than that your code looks good to merge

File: components/navbar/navbar.css (hunk 1 above)
Comment (MurtazaBagwala07) [reply components/navbar/navbar.css]: yes i want it responsive when i use it in documentation

Comment (MurtazaBagwala07) [reply components/navbar/navbar.css]: yes will do

**Pull request merged**